content/post/087-js-ipfs-0.42.md

```diff
@@ -0,0 +1,196 @@
+---
+date: 2020-04-14
+url: 2020-04-14-js-ipfs-0-42
+title: js-ipfs 0.42.0 released
+author: Alex Potsides
+---
+
+# 🔦 Highlights
+
+> Smaller, faster, more reliable
+
+`js-IPFS@0.42.0` has been released, bringing a host of bug fixes and other small stability improvements.
+
+It reduces the size of the minified bundle by 7.5% for faster downloads and more responsive web apps!
+
+We've also merged some of our repos together, our hope is that this will make contributing easier and encourage experimentation as there will be fewer PRs to juggle to land a piece of functionality!
```

Comment:
```suggestion
We've also merged some of our repos together which were logically intertwined. Our hope is that this will make contributing easier and encourage experimentation as there will be fewer PRs to juggle to land a piece of functionality!
```